.storybook/config.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-env node */
```

Comment:
This shouldn't be necessary: https://github.com/sqren/apm-components/blob/master/.eslintrc.json#L12